src/test/rgw/bucket_notification/api.py

```diff
@@ -242,17 +242,21 @@ def admin(args, cluster='noname', **kwargs):
     cmd = [test_path + 'test-rgw-call.sh', 'call_rgw_admin', cluster] + args
     return bash(cmd, **kwargs)
 
-def delete_all_topics(conn, tenant, cluster):
+def delete_all_topics(conn, tenant, cluster, version = None):
     """ delete all topics """
     if tenant == '':
         topics_result = admin(['topic', 'list'], cluster)
         topics_json = json.loads(topics_result[0])
+        if(version == 'v1'):
```

Comment:
this may break, depending with the order opf execution of the test.
what if you try to delete v1 topics that were created intentionally in v1, whiule the conf says v2?

would recommend a more robust way, do a try-catch on Key error, and select the option based on that